src/components/Form/FormSection.jsx

```diff
@@ -14,28 +18,51 @@ import Typography from '@mui/material/Typography';
  * that consist of title and children (see {@link formSectionProps})
  * @param {string} Props.headingId - The `id` attribute of the element containing the `title`
  * @param {string} Props.title - Title of form section
- * @param {React.ReactElement} Props.children - JSX Element of the wrapped form
- * @returns {React.JSX.Element} - The FormSection Component
+ * @param {React.ReactElement} Props.children - The child elements to be rendered inside the main content area
+ * @returns {React.JSX.Element} The FormSection Component
  */
-const FormSection = ({ title, headingId, children }) => (
-  <Box
-    sx={{
-      marginTop: 1,
-      display: 'flex',
-      flexDirection: 'column',
-      justifyContent: 'center',
-      alignItems: 'center',
-      mx: '3px',
-      padding: '16px',
-      minWidth: '50%',
-      maxWidth: '75dvw'
-    }}
-  >
-    <Typography variant="h2" align="center" mb={2} sx={{ fontSize: '20px' }} id={headingId}>
-      {title}
-    </Typography>
-    {children}
-  </Box>
-);
+const FormSection = ({ title, headingId, children }) => {
+  const theme = useTheme();
+  // eslint-disable-next-line no-unused-vars
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+
+  const truncatedText = title ? truncateText(title) : '';
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        padding: '20px',
+        // minWidth: '50vw',
+        // minWidth: isSmallScreen ? null : '50dvw',
+        // maxWidth: isSmallScreen ? '100dvw' : '75dvw',
+        width: '100%'
+      }}
+    >
+      <Typography
+        variant="h2"
+        align="center"
+        mb={1}
+        sx={{
+          fontSize: '20px',
+          fontWeight: 'bold',
+          // TODO: Discuss whether any widths (normal, max, and/or min) should be set
+          // width: isSmallScreen ? '100dvw' : '50dvw',
+          // whiteSpace: 'nowrap',
+          // overflow: 'hidden',
+          // textOverflow: 'ellipsis',
+          maxWidth: '100%'
```

Comment:
We'll probably want to set some sort of minimum width or fixed width at least when it's still in Desktop mode. Otherwise, when the text file name is too short, the modal shrinks to the width of the element with the filename.

<img width="228" alt="Screenshot 2024-05-21 at 5 53 30 PM" src="https://github.com/codeforpdx/PASS/assets/14917816/c5eb1b4f-5490-413f-a3bf-a238d3bc7029">

Reply:
Yeah this one I was least decided-upon so I left a bunch of possible ideas commented out. I don't have a super strong opinion on how to implement a width currently. But I do think there should be one.

Reply:
Think we can just set a min width of 300px here then

Reply:
Easily done.

The only qualm is that for some reason the `NewMessageModal` reacts strangely to set widths. It has to do with the `oldMessage` conditional starting around line 183 that converts it from a new message into a reply.

<img width="388" alt="2024-05-22 (4)" src="https://github.com/codeforpdx/PASS/assets/83156697/848a1858-3dd3-4113-b159-509197d4d939">
<img width="395" alt="2024-05-22 (5)" src="https://github.com/codeforpdx/PASS/assets/83156697/4959d244-696d-41ef-8b2b-d5259977c8b5">

Reply To is formatted the way all the other modals are. It's just New Message that squashes the content.

Reply:
Got it.

Think we can have that problem with the reply modal addressed in a new issue.